pkgs/by-name/wi/wivrn/package.nix

```diff
@@ -127,18 +104,41 @@ stdenv.mkDerivation (finalAttrs: {
     x264
   ] ++ lib.optionals cudaSupport [
     cudaPackages.cudatoolkit
-  ];
+  ] ++ monado.buildInputs;
 
   cmakeFlags = [
     (lib.cmakeBool "WIVRN_BUILD_CLIENT" false)
     (lib.cmakeBool "WIVRN_USE_VAAPI" true)
     (lib.cmakeBool "WIVRN_USE_X264" true)
     (lib.cmakeBool "WIVRN_USE_NVENC" cudaSupport)
+    (lib.cmakeBool "WIVRN_USE_SYSTEMD" true)
+    (lib.cmakeBool "WIVRN_USE_PIPEWIRE" true)
+    (lib.cmakeBool "WIVRN_USE_PULSEAUDIO" true)
```

Comment:
Are you sure that both can be used?

Reply:
Yup. This just builds support for it; if the audio backend isn't present, it just doesn't use it.